tests/integration/records/records_schema_v1_schema.json

```diff
@@ -1,294 +1,58 @@
 {
```

Comment:
This is a weird way to validate whether a json is correctly formatted but is the way that we're currently doing it.